cdbsearch.py

```diff
@@ -387,16 +384,7 @@ def cdbsearch(epd, depthLimit, concurrency, evalDecay):
             int(1000 * runtime / chessdb.count_uncached.get()),
         )
 
-        pvline = ""
```

Comment:
the pv line returned by search could currently end on checkmate or draw. This loop was to remove these things. However, maybe it is better not return checkmate or draw in the PV line.

Reply:
oh, sorry. I completely missed that. I agree it would probably be best to stick to pure cdb syntax for the pv, i.e. uci moves only. do you want me to make these changes in my branch and then push to this PR?

Reply:
Or we can leave those and just manually filter for them here when we create URL. that would be the minimalist change and keep the info on screen for users.